vendors/nuvoton/boards/numaker_iot_m487_wifi/aws_tests/application_code/main.c

```diff
@@ -56,7 +56,17 @@
 #define mainLOGGING_TASK_STACK_SIZE         ( configMINIMAL_STACK_SIZE * 5 )
 #define mainLOGGING_MESSAGE_QUEUE_LENGTH    ( 15 )
 
-#define mainTEST_RUNNER_TASK_STACK_SIZE     ( configMINIMAL_STACK_SIZE * 20 )
+#define mainTEST_RUNNER_TASK_STACK_SIZE     ( configMINIMAL_STACK_SIZE * 40 )
+
+#ifdef IOT_NETWORK_RECEIVE_TASK_PRIORITY
+
+/* Keeping the test runner task priority higher than that of the Network receive task
+ * priority so as to make sure that the receive of the incoming packets are only happening
+ * after successfully updating the internal data structures in test runner task for a send. */
+    #define mainTEST_RUNNER_TASK_PRIORITY    ( IOT_NETWORK_RECEIVE_TASK_PRIORITY + 1 )
```

Comment:
Is there any possibility of this overflowing?

Reply:
Yea. That is a good point. Added an assert to make sure that. Thanks